extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -143,6 +143,27 @@ public function getTopAdminsData() {
 	}
 
 	/**
+	 * Set context for allAdmins template. Needs to be passed through the index method in order to work.
+	 * @return array
+	 */
+	public function getAllAdminsData() {
+		$allAdmins = CommunityPageSpecialUsersModel::filterGlobalBots(
+			// get all admins who have contributed in the last two years ordered by contributions
+			CommunityPageSpecialUsersModel::getTopContributors( 10, false, true )
```

Comment:
I think this is a leftover from the code initially written. I can't really see any reason why those methods should remain static, so I'll change it.